### PR TITLE
#166240947 Implements password reset

### PR DESCRIPTION
**Pull request by mifeille**
#### What does this PR do?
This PR will enable users to reset their password

#### Description of Task to be completed?
- Adds reset password forms
- Adds a reset password link to the login page
- Adds redux logic
- Tests components, actions, and reducers

#### How should this be manually tested?
This PR can be tested by cloning this branch and running the development server with this command `yarn dev`

#### Screenshots (if appropriate)
ezgif com-video-to-gif (9)

#### What are the relevant pivotal tracker stories?
#166240947

### CODE REVIEW

File: src/components/pages/ResetPasswordEmail.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+import resetPassword from '../../redux/actions/resetPassword.actions';
+import '../../assets/scss/pages/_resetPasswordEmail.scss';
+
+export class ResetPasswordEmail extends Component {
+  state = {
+    email: '',
+    errorMessage: '',
+  }
+
+  componentDidMount() {
```
Comment (deschantkn) [inline src/components/pages/ResetPasswordEmail.js]: Hello @mifeille, would you kindly delete this unused method?

**Review by deschantkn — APPROVED** [review]

File: src/components/pages/ApplyPasswordForm.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+  render() {
+    const {
+      newpassword, confirmPassword, errorMessage, passwordError,
+    } = this.state;
+    // eslint-disable-next-line react/prop-types
+    const { displayMsg, status } = this.props;
+    let inputPassword = (
+      <form onSubmit={this.handleSubmit}>
+        <div className="field">
+          <input
```
Comment (eliemugenzi) [inline src/components/pages/ApplyPasswordForm.js]: We have reusable TextInput components, check them out.

File: src/index.js [hunk 3]
```diff
@@ -5,6 +5,7 @@ import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
 import store from './redux/store';
 import App from './components/App';
+import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
```
Comment (eliemugenzi) [inline src/index.js]: This is no longer necessary since we have Bootstrap imported in our HTML file.

**Review by eliemugenzi — CHANGES_REQUESTED** [review]

**Review by eliemugenzi — APPROVED** [review]
Looks Good to Approve (LGTA). Good job @mifeille.

**Review by EmyRukundo — APPROVED** [review]

**Pull request merged**